locales/hi.json

```
@@ -1,1542 +1,1542 @@
+  "Click this button to expand or collapse the auxillary toolbar.": "सहायक उपकरण पट्टी को विस्तृत या संक्षिप्त करने के लिए यह बटन क्लिक करें.",
+  "Displays help messages for the main and auxiliary toolbar, right-click contextual menu for blocks and canvas, and palettes.": "मुख्य और सहायक टूलबार, ब्लॉक्स और कैनवास के राइट-क्लिक संदर्भ मेनू, और पैलेट्स के लिए सहायता संदेश दिखाता है।",
+  "Click to run the project in slow mode.": "प्रोजेक्ट को धीरे चलाने के लिए यहाँ क्लिक करें",
+  "Click to run the project step by step.": "प्रोजेक्ट को क्रमशः चलने के लिए क्लिक करें",
+  "Display statistics about your Music project.": "अपने संगीत प्रोजेक्ट के आँकड़े दिखाएँ।",
+  "Load a selected plugin.": "चयनित प्लगइन लोड करें।",
+  "Delete a selected plugin.": "चयनित प्लगइन को हटाएं।",
+  "Enable scrolling": "स्क्रॉलिंग सक्षम करें",
+  "You can scroll the blocks on the canvas.": "आप कैनवास पर ब्लॉक्स को स्क्रॉल कर सकते हैं।",
+  "Switch between dark and light mode.": "डार्क और लाइट मोड के बीच स्विच करें।",
+  "Click to add another project into the current one.": "वर्तमान प्रोजेक्ट में एक और प्रोजेक्ट जोड़ने के लिए क्लिक करें।",
+  "Wrap Turtle": "टर्टल को लपेटें",
+  "Turn Turtle wrapping On or Off.": "टर्टल लपेटें को चालू या बंद करें।",
+  "Click to set the current pitch.": "वर्तमान पिच सेट करने के लिए क्लिक करें।",
+  "Converts Music Block programs to JavaScript.": "म्यूजिक ब्लॉक प्रोग्राम को जावास्क्रिप्ट में परिवर्तित करता है।",
+  "Restore blocks from the trash.": "कचरे से ब्लॉक्स को बहाल करें।",
+  "Switch mode": "मोड स्विच करें",
+  "Switch between beginner and advance modes.": "शुरुआती और उन्नत मोड के बीच स्विच करें।",
+  "Select your language preference.": "अपनी भाषा प्राथमिकता का चयन करें।",
+  "Contextual Menu for Blocks": "ब्लॉक्स के लिए संदर्भ मेनू",
+  "Right-click on a block to access common actions.": "सामान्य क्रियाओं तक पहुंचने के लिए ब्लॉक पर राइट-क्लिक करें।",
+  "Delete": "हटाएं",
+  "To delete a block, right-click on it. Then you will see the delete option": "To delete a block, right-click on it. Then you will see the delete option",
+  "To copy a block, right-click on it. Then you will see the copy option": "To copy a block, right-click on it. Then you will see the copy option",
+  "To extract a block, right-click on it. Then you will see the extract option": "To extract a block, right-click on it. Then you will see the extract option",
+  "Keyboard shortcuts": "कीबोर्ड शॉर्टकट्स",
+  "You can type d to create a do block and r to create a re block etc.": "आप डू ब्लॉक बनाने के लिए डी और रे ब्लॉक बनाने के लिए आर इत्यादि टाइप कर सकते हैं।",
+  "Contextual Menu for Canvas": "कैनवास के लिए संदर्भ मेनू",
+  "Right-click on the canvas to perform additional canvas-related tasks.": "अतिरिक्त कैनवास संबंधित कार्य करने के लिए कैनवास पर राइट-क्लिक करें।",
+  "Turn on/off lines for cartesian or polar grids.": "कार्टेशियन या पोलर ग्रिड की लाइनों को चालू/बंद करें।",
+  "Turn on/off music staffs.": "म्यूजिक स्टाफ को चालू/बंद करें।",

```

since staffs is in plural
shouldn't we also wright स्टाफों ?
```suggestion
  "Turn on/off music staffs.": "म्यूजिक स्टाफों  को चालू/बंद करें।",
```